requirements.txt

```diff
@@ -5,6 +5,6 @@ pytest==2.7.3
 pytest-gitignore==1.3
 pytest-mock==0.9.0
 pytest-pylint==0.4.0
-requests>=2.9.1,<3
+requests
```

Comment:
Unfortunately it doesn't promise compatibility. Generally, I've avoided pinning in library packages unless there's a known version of a dependency that breaks things.  This allows apps to pin their dependencies as needed.

Funnily enough, this is how we found this issue in the first place. We had an internal package pinning `tldextract>1.2` and an internal application pinning `requests==2.2`. `tldextract` auto-updated on a release and we ran in to a hell of version conflicts.

I just did another test run locally with requests==2.0.1 and it failed. I'll bump the pin to be >=2.1.0 since that's the lowest known working version. I'm of the opinion that leaving the upper-end unrestricted (i.e. don't pin requests<3) is alright, since it's not yet known to fail.